Client/Application/SceneDelegate.swift

```diff
@@ -117,19 +117,6 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // reached via a "Spotlight" search before we began indexing visited pages via CoreSpotlight.
         if let url = userActivity.webpageURL {
             let query = url.getQuery()
```

Comment:
@lmarceau do you need this then?

Reply:
Am I looking at this wrong or the value of `query` is not used 🤔

Reply:
Yeah you're right! Missed that, let me remove it.